go.work.sum

```diff
@@ -1,17 +1,406 @@
+4d63.com/gochecknoglobals v0.1.0/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
```

Comment:
I wouldn't expect any of the go dependencies to change as part of this PR

Reply:
yeah, running `make dist` changes it for me. This happens even on main with the latest commit. I have go 1.18.1 on my system.